docs/src/increase-observability/metrics/dedicated-generation-3.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Monitor Dedicated Generation 3 metrics
+sidebarTitle: Dedicated Generation 3 metrics
+description: Understand how to read metrics for Dedicated Genration 3 environments.
+---
+
+Dedicated Generation 3 environments consist of various containers running across dedicated virtual machines:
+
+* App containers: one or more [app containers](../../create-apps/_index.md)
+* Service containers: zero or more [service containers](../../add-services/_index.md)
+* Worker containers: zero or more [worker instances](../../create-apps/app-reference.md#workers).
+
+Infrastructure metrics report CPU, RAM, and disk space for all containers.
+
+By default, the graphs include all hosts and an average over the hosts.
+To select metrics for specific hosts, click **Filter**.
+
+![Clicking Filter reveals a list of hosts you can filter](/images/metrics/filtering-gen3.png "0.4")
+
+The ID numbers for the hosts in the list for filtering match the numbers for interacting with a host,
+such as for accessing the environment using SSH.
+
+## Example of how to read metrics
+
+This example should give you an idea of how the metrics appear.
+Dedicated Generation 3 environments metrics show resource usage for each app, service, and worker container
+across all hosts.
+
+This reference project has a single app, two services (MySQL and Redis), and one worker.
+
+Once you've read the metrics, see [recommendations for action](./_index.md#dedicated-generation-3-environments).
+
+### App container
+
+Metrics graphs for the app container show CPU, RAM, and disk allocation and usage.
```

Comment:
:pick: :arrow_forward: 

Shouldn't a [serial comma](https://developers.google.com/style/commas?hl=en) be used (also to avoid the repetition of and).
```suggestion
Metrics graphs for the app container show CPU, RAM, disk allocation, and usage.
```

Reply:
There are only 3 items in the list: CPU, RAM, and disk. They are all used as attributes for `allocation and usage` together. It's the same as we currently have: https://github.com/platformsh/platformsh-docs/blob/main/docs/src/increase-observability/metrics/grid.md?plain=1#L49

Reply:
Thanks for the clarification there! My bad :)